pkg/groupie/groups.go

```diff
@@ -215,8 +218,10 @@ var (
 		"XSS vulnerability in the HTML parser":                                                       "javascript",
 		"jQuery before 3.4.0, as used in Drupal, Backdrop CMS, and other products, mishandles jQuery.extend(true, {}, ...) because of Object.prototype pollution": "javascript",
 		"A prototype pollution vulnerability in handlebars is exploitable if an attacker can control the template":                                                "javascript",
-		"XSS":                 "javascript",
-		"Prototype pollution": "javascript",
+		"XSS":                             "javascript",
+		"Prototype pollution":             "javascript",
+		"JQuery 1.2 < 3.5.0 Multiple XSS": "javascript",
+		"Regex in its jQuery.htmlPrefilter  sometimes may introduce XSS": "javascript",
```

Comment:
Is there an extra space between `jQuery.htmlPrefilter` and `sometimes`?

Reply:
Yes, you are right! Good catch :)